### PR TITLE
Mesh docs

### PR DESCRIPTION
**Pull request by ruromero**
Updated the Kong Mesh instructions with the steps to deploy:

* Control plane
* Metrics
* Tracing
* Logging
* Kuma-demo (instead of kuma-counter-demo)

### CODE REVIEW

File: kong-mesh.md [hunk 1]
```diff
@@ -3,89 +3,236 @@
...
 ```
 
-- Verify the Installation
-```
-oc port-forward svc/kong-mesh-control-plane -n kong-mesh-system 5681:5681
+b) Using the internal registry (avoid docker.io). See [using the openshift-registry](./openshift-registry/README.md)
+
+```bash
+KONG_REGISTRY=$(oc get route default-route -n openshift-image-registry --template='{{ .spec.host }}')/kong-image-registry
+./kumactl install control-plane --dataplane-registry=$KONG_REGISTRY --control-plane-registry=$KONG_REGISTRY --cni-enabled --license-path=./license.json  | kubectl apply -f -
```
Comment (mpaulgreen) [inline kong-mesh.md:18]: @ruromero we still get the error for pulling the images from internal registry. Yesterday @cmwylie19 got the same error. 
``` Warning  Failed          10s (x3 over 48s)  kubelet            Failed to pull image "default-route-openshift-image-registry.apps.mpkongdemo.51ty.p1.openshiftapps.com/kong-image-registry/kuma-cp:1.7.0": rpc error: code = Unknown desc = pinging container registry default-route-openshift-image-registry.apps.mpkongdemo.51ty.p1.openshiftapps.com: Get "https://default-route-openshift-image-registry.apps.mpkongdemo.51ty.p1.openshiftapps.com/v2/": x509: certificate signed by unknown authority```

File: kong-mesh.md [hunk 2]
```diff
@@ -3,89 +3,236 @@
...
 ```
 
-- Apply scc  of non-root to ```kuma-metrics```
+- Expose the service
+
+```{bash}
+oc expose svc/kong-mesh-control-plane -n kong-mesh-system
```
Comment (mpaulgreen) [inline kong-mesh.md:24]: The targetPort of the route is not always set correctly

Comment (ruromero) [reply kong-mesh.md:24]: what do you mean?

Comment (mpaulgreen) [reply kong-mesh.md:24]: The route always do not point to targetPort of ```http-api-server``` of the service. We may to validate it and path the route if it is correctly not set.

Comment (ruromero) [reply kong-mesh.md]: Thanks for noticing it, it's fixed

File: kong-mesh.md (hunk 1 above)
Comment (ruromero) [reply kong-mesh.md:18]: Added instructions for trusting the external route internally

**Pull request merged**